core/src/data_sources/splitter.rs

```diff
@@ -481,15 +482,20 @@ impl Splitter for BaseV0Splitter {
         max_chunk_size: usize,
         section: Section,
     ) -> Result<Vec<String>> {
+        let start = Instant::now();
+
         let mut embedder = provider(provider_id).embedder(model_id.to_string());
         embedder.initialize(credentials).await?;
 
+        let duration = start.elapsed();
```

Comment:
Yeah this line is not well positioned

I undertstand now what you said embedding it's just because we load the embeddder model but we only use it to tokenizer here :+1:

Reply:
Yeah figure this out. I'll close this PR.

Reply:
I'll find another way to instrument all embedding operations.

Reply:
adding some more instrumentation here: https://github.com/dust-tt/dust/pull/3198